<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorWindow</class>
 <widget class="QMainWindow" name="EditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tTexture</string>
  </property>
  <widget class="QWidget" name="MainWindow">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>12</y>
      <width>861</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="MainWindowLayout">
     <item>
      <layout class="QHBoxLayout" name="ConversionTypeSectionLayout">
       <item>
        <widget class="QLabel" name="ConversionTypeLabel">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Conversion Type</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>150</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="ConversionTypeLayout">
         <item>
          <layout class="QVBoxLayout" name="Texture2DLayout">
           <item>
            <widget class="QPushButton" name="Texture2DButton">
             <property name="text">
              <string>Texture2D</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Texture2DLabel">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Texture 2D</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="HCrossLayout">
           <item>
            <widget class="QPushButton" name="HCrossButton">
             <property name="text">
              <string>H-Cross</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="HCrossLabel">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>H-Cross</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="VCrossLayout">
           <item>
            <widget class="QPushButton" name="VCrossButton">
             <property name="text">
              <string>V-Cross</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="VCrossLabel">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>V-Cross</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="EquirectangularLayout">
           <item>
            <widget class="QPushButton" name="EquirectangularButton">
             <property name="text">
              <string>Equirectangular</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="EquirectangularLabel">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Equirectangular</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="InputLabel">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Input</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="InputLayout">
         <item>
          <layout class="QHBoxLayout" name="InputFilepathLayout">
           <item>
            <widget class="QLabel" name="InputFilepathLabel">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Filepath</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="InputFilepathValue">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="InputFilepathButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="InputChannelLayout">
           <item>
            <widget class="QLabel" name="InputChannelLabel">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Channels</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="InputChannelBox">
             <property name="editable">
              <bool>false</bool>
             </property>
             <item>
              <property name="text">
               <string>3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="FlipOnLoadLayout">
           <item>
            <widget class="QCheckBox" name="FlipOnLoadCheckbox">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>1677215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>FlipOnLoad</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="OutputSectionLayout">
       <item>
        <widget class="QLabel" name="OutputLabel">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Output</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="OutputLayout">
         <item>
          <layout class="QHBoxLayout" name="OutputFilepathLayout">
           <item>
            <widget class="QLabel" name="OutputFilepathLabel">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Filepath</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="OutputFilepathValue">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="OutputFilepathButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="PrefilterLayout">
           <item>
            <widget class="QCheckBox" name="PrefilterCheckbox">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>Prefilter</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="PrefilterSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="FooterLayout">
       <item>
        <layout class="QHBoxLayout" name="ExportLayout">
         <item>
          <widget class="QLabel" name="ExportLabel">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Export</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ExportValue">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="ButtonBox">
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>887</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
